packages/app/src/app/hooks/useWorkspaceSubscription.ts

```diff
@@ -50,6 +50,7 @@ export const useWorkspaceSubscription = (): WorkspaceSubscriptionReturn => {
     return {
       ...NO_SUBSCRIPTION,
       isEligibleForTrial: isTeamSpace, // Currently, only teams are eligible for trial.
+      numberOfSeats: activeTeamInfo.limits.maxEditors,
```

Comment:
We can remove the `numberOfSeats` in `NO_SUBSCRIPTION` if we want or we need to update the object and the types to reflect the `maxEditors` type.